lib/archethic/p2p/message.ex

```diff
@@ -208,8 +209,11 @@ defmodule Archethic.P2P.Message do
         welcome_node_public_key: welcome_node_public_key,
         validation_node_public_keys: validation_node_public_keys
       }) do
+    encoded_validation_node_public_keys_length =
```

Comment:
As this array cannot overpass a length of 200 we don't need to use VarInt there

Reply:
Changed for length(nb_validation_nodes)